qa/L0_infer/test.sh

```diff
@@ -38,6 +38,8 @@ if [ ! -z "$TEST_REPO_ARCH" ]; then
     REPO_VERSION=${REPO_VERSION}_${TEST_REPO_ARCH}
 fi
 
+ldconfig || true
```

Comment:
Is this for iGPU?

Reply:
Yes, this if for iGPU but shouldn't impact regular containers, as it fixing the dynamic linking.